tools/macos-cleanup.sh

```diff
@@ -52,4 +52,10 @@ echo 'Cleanup any old versions of gems…'
 gem cleanup &>/dev/null
 
 echo 'Purge inactive memory…'
-sudo purge
\ No newline at end of file
+sudo purge
+
+echo 'Remove Jenkins error logs…'
+rm ~/jenkins_err.log
+
+echo 'Remove workspace, it will be recreated again…'
+rm -rfv ~/build/workspace/*
```

Comment:
Since the script is executed as root, shouldn't these be absolute with `/Users/iojs/` ?

Reply:
It will be more correct to use absolute path, I will try that in a momment on `release-orka-macos11-x64-1` and double check 👍

Reply:
I tested in the machine and it is working fine, so I added the change in 62bcfd11797e6f3b1d50cbb97df09713effeaa2b